package.json

```diff
@@ -63,5 +64,8 @@
         "statements": 100
       }
     }
+  },
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0"
```

Comment:
idk if i like the addition of a dependency. this project has never had one before, and it'd be nice to keep it really lightweight. i'm fine if we have to drop browser support, but i don't want the bundle to bloat.

See: https://bundlephobia.com/result?p=react-detect-offline@2.1.2. Notice how 100% of the size is "self". Your dependency is https://bundlephobia.com/result?p=@babel/polyfill@7.2.5. That would change the bundle from 1.7kB to 27.4kb + 1.7kb (gzipped). That's a 1600% increase :)

Reply:
the polyfill shouldn't be a dep, it should probably just compile away.